### PR TITLE
Add Swedish translation

### PR DESCRIPTION
**Pull request by olleolleolle**
This PR adds a Swedish translation file based on the Norwegian one.

## Question for reviewer

If you know Swedish, please do take a look, see if I made any typos or otherwise were unclear.

### CODE REVIEW

File: locales/sv-SE.json [hunk 1]
```diff
@@ -0,0 +1,336 @@
...
+    "Fallback captions: ": "Ersättningsundertexter: ",
+    "Show related videos: ": "Visa relaterade videor? ",
+    "Show annotations by default: ": "Visa länkar-i-videon som förval? ",
+    "Visual preferences": "Visuella inställningar",
+    "Player style: ": "Spelarstil: ",
+    "Dark mode: ": "Mörkt läge: ",
+    "Theme: ": "Tema: ",
+    "dark": "Mörk",
+    "light": "Ljus",
+    "Thin mode: ": "Tunt läge: ",
```
Comment (quite) [inline locales/sv-SE.json]: `Lättviktigt läge` (light-wight mode) would be better, but that's not what it says in English. The mode omits images

File: locales/sv-SE.json [hunk 2]
```diff
@@ -0,0 +1,336 @@
...
+    "Subscription preferences": "Prenumerationsinställningar",
+    "Show annotations by default for subscribed channels: ": "Visa länkar-i-videor som förval för kanaler som prenumereras på? ",
+    "Redirect homepage to feed: ": "Omdirigera hemsida till flöde: ",
+    "Number of videos shown in feed: ": "Antal videor att visa i flödet: ",
+    "Sort videos by: ": "Sortera videor på: ",
```
Comment (quite) [inline locales/sv-SE.json]: `Sortera videor: `

File: locales/sv-SE.json [hunk 3]
```diff
@@ -0,0 +1,336 @@
...
+    "published": "publicerad",
+    "published - reverse": "publicerad - omvänd",
```
Comment (quite) [inline locales/sv-SE.json]: `publicering - omvänd`

File: locales/sv-SE.json [hunk 4]
```diff
@@ -0,0 +1,336 @@
...
+    "Theme: ": "Tema: ",
+    "dark": "Mörk",
+    "light": "Ljus",
+    "Thin mode: ": "Tunt läge: ",
+    "Subscription preferences": "Prenumerationsinställningar",
+    "Show annotations by default for subscribed channels: ": "Visa länkar-i-videor som förval för kanaler som prenumereras på? ",
+    "Redirect homepage to feed: ": "Omdirigera hemsida till flöde: ",
+    "Number of videos shown in feed: ": "Antal videor att visa i flödet: ",
+    "Sort videos by: ": "Sortera videor på: ",
+    "published": "publicerad",
```
Comment (quite) [inline locales/sv-SE.json]: `publicering`

File: locales/sv-SE.json [hunk 5]
```diff
@@ -0,0 +1,336 @@
...
+    "alphabetically - reverse": "alfabetiskt - omvänd",
+    "channel name": "kanalnamn",
+    "channel name - reverse": "kanalnamn - omvänd",
+    "Only show latest video from channel: ": "Visa bara senaste videon från kanal: ",
+    "Only show latest unwatched video from channel: ": "Visa bara senaste osedda videon från kanal: ",
+    "Only show unwatched: ": "Visa bara osedda: ",
+    "Only show notifications (if there are any): ": "Visa endast aviseringar (om det finns några): ",
+    "Enable web notifications": "Slå på aviseringar",
+    "`x` uploaded a video": "`x` laddade upp en video",
+    "`x` is live": "`x` är live",
```
Comment (quite) [inline locales/sv-SE.json]: `x` sänder live

(or is that old-fashioned? :)

File: locales/sv-SE.json [hunk 6]
```diff
@@ -0,0 +1,336 @@
...
+    "revoke": "återkalla",
+    "Subscriptions": "Prenumerationer",
+    "`x` unseen notifications": "`x` osedda aviseringar",
+    "search": "sök",
+    "Log out": "Logga ut",
+    "Released under the AGPLv3 by Omar Roth.": "Utgiven under AGPLv3-licens av Omar Roth.",
+    "Source available here.": "Källkod tillgänglig här.",
+    "View JavaScript license information.": "Visa JavaScript-licensinformation.",
+    "View privacy policy.": "Visa privatlivspolicy.",
+    "Trending": "Trendsättende",
```
Comment (quite) [inline locales/sv-SE.json]: `Trendar`? eller `Trendsättande` då (också i senare fras)

File: locales/sv-SE.json [hunk 7]
```diff
@@ -0,0 +1,336 @@
...
+    "Wrong answer": "Fel svar",
+    "Erroneous CAPTCHA": "Ogiltig CAPTCHA",
+    "CAPTCHA is a required field": "CAPTCHA är ett obligatoriskt fält",
+    "User ID is a required field": "Användar-ID är ett obligatoriskt fält",
+    "Password is a required field": "Lösenord är ett obligatoriskt fält",
+    "Wrong username or password": "Ogiltigt användarnamn eller lösenord",
+    "Please sign in using 'Log in with Google'": "Logga in genom \"Google-inloggning\"",
+    "Password cannot be empty": "Lösenordet kan inte vara tomt",
+    "Password cannot be longer than 55 characters": "Lösenordet kan inte vara längre än 55 tecken",
+    "Please log in": "Logga inn",
```
Comment (quite) [inline locales/sv-SE.json]: `Logga in`

File: locales/sv-SE.json [hunk 8]
```diff
@@ -0,0 +1,336 @@
...
+    "Invidious Private Feed for `x`": "Ogiltig privat flöde för `x`",
+    "channel:`x`": "kanal `x`",
+    "Deleted or invalid channel": "Raderad eller ogiltig kanal",
+    "This channel does not exist.": "Denna kanal finns inte.",
+    "Could not get channel info.": "Kunde inte hämta kanalinfo.",
+    "Could not fetch comments": "Kunde inte hämta kommentarer",
+    "View `x` replies": "Visa `x` svar",
+    "`x` ago": "`x` sedan",
+    "Load more": "Ladda flera",
```
Comment (quite) [inline locales/sv-SE.json]: `Ladda fler`

File: locales/sv-SE.json [hunk 9]
```diff
@@ -0,0 +1,336 @@
...
+    "Not a playlist.": "Ogiltig spellista.",
+    "Playlist does not exist.": "Spellistan finns inte.",
+    "Could not pull trending pages.": "Kunde inte hämta trendsättande sidor.",
+    "Hidden field \"challenge\" is a required field": "Dolt fält \"challenge\" är ett obligatoriskt fält",
+    "Hidden field \"token\" is a required field": "Dolt fält \"token\" är ett obligatoriskt fält",
+    "Erroneous challenge": "Felaktig challenge",
+    "Erroneous token": "Felaktig token",
+    "No such user": "Ogiltig användare",
+    "Token is expired, please try again": "Token föråldrad, försök igen",
+    "English": "Engelska",
```
Comment (quite) [inline locales/sv-SE.json]: In Swedish all the following names of languages are written in all lower case ;)

File: locales/sv-SE.json (hunk 1 above)
Comment (olleolleolle) [reply locales/sv-SE.json]: "Läge utan bilder" or "Visa Inga bilder" or something?

File: locales/sv-SE.json (hunk 9 above)
Comment (olleolleolle) [reply locales/sv-SE.json]: I believe they're presented in a language picker, each its own option.

Do you mean: "We ought to turn them into lower-case variants" for that list?

**Comment by quite** [comment]
I think we stay with the translation from English, until that phrase changes...

**Comment by quite** [comment]
I think I would have changed it. For me they are just ending a sentence. But, really doesn't matter.

**Comment by olleolleolle** [comment]
Thanks for the extra eyes, I ditched the translations for the languages, probably hurts more than it helped.

**Comment by omarroth** [comment]
Looks great! Thank you both so much!

**Pull request merged**